hello_sqlite/config/host.exs

```diff
@@ -0,0 +1,14 @@
+import Config
+
+# Add configuration that is only needed when running on the host here.
+
+config :hello_sqlite,
+  ecto_repos: [HelloSqlite.Repo]
+
+config :hello_sqlite, HelloSqlite.Repo,
+  database: "/tmp/database.db",
+  show_sensitive_data_on_connection_error: false,
+  journal_mode: :wal,
+  cache_size: -64000,
```

Comment:
Is it ok that the cache has a negative size?

Reply:
yep. see [the docs](https://hexdocs.pm/ecto_sqlite3/Ecto.Adapters.SQLite3.html#module-provided-options)
(i added those docs to the example as well)